pkgs/development/libraries/tbb/default.nix

```diff
@@ -62,6 +65,12 @@ stdenv.mkDerivation rec {
       --replace 'conformance_resumable_tasks' ""
   '';
 
+  cmakeFlags = [
+    (lib.cmakeBool "BUILD_SHARED_LIBS" (!static))
```

Comment:
I think we have something like `stdenv.isStatic` or similar around in the code, you can consider using it for that.

Reply:
I've changed it to `static ? stdenv.hostPlatform.isStatic` which seems to be the most common way of doing things if you grep this repo for `static \?`. Note tbb doesn't actually build via linux `pkgsStatic` yet for a reason that IMO is out of scope of this PR:
```
tbb-static-x86_64-unknown-linux-musl> CMake Error at test/CMakeLists.txt:29 (add_executable):
tbb-static-x86_64-unknown-linux-musl>   The target name "conformance/.cpp" is reserved or not valid for certain
tbb-static-x86_64-unknown-linux-musl>   CMake features, such as generator expressions, and may result in undefined
tbb-static-x86_64-unknown-linux-musl>   behavior.
```